src/regridStates/gauIncr2native.nml

```diff
@@ -0,0 +1,31 @@
+&config
+ n_vars=4,                             ! number of vars in list below
+ variable_list(1)="soilt1_inc     ", 
+ variable_list(2)="soilt2_inc     ", 
+ variable_list(3)="slc1_inc     ", 
+ variable_list(4)="slc2_inc     ", 
+ missing_value=0.,                     ! any un-filled grid cell takes this value. 
+                                       ! use 0. for increments
+/
+
+! required fields depend on gridtype, and whether input or output 
+&input
+ gridtype="gau_inc",                         ! "gau_inc" for GSI/Gaussian increment file 
+ ires=192,
+ jres=96,
+ fname="enkfgdas.t12z.sfci003.ensmean.nc",
+ dir="./input/",
+ fname_coord="gaussian.192.96.nc", ! this is the scrip file generated by ufs_utils weight_gen
+ dir_coord="./input/"
+/
+
+&output
+ gridtype="fv3_rst",    ! "fv3_rst" any native model grid file (increment or restart)
+ ires=48,
+ jres=48,
+ fname="inc2ens.sfc_data",
+ dir="./output/",
+ fname_mask="20211221.090000.sfc_data"
```

Comment:
Is this template for mask? as in mask will be written in this file format?

Reply:
This is the file that is read in to get the mask. I changed the code to use a fix file instead of a restart though - so I need to change the entry here too. I'll also add a block of comments explaining each variable.

Reply:
Changed this file, and added comment block at the end of the example namelist.